npu.user.js

```diff
@@ -985,9 +989,43 @@ var npu = {
 		
 	/* == EXAM LIST == */
 	
+		/* Decide whether the given grade string constitutes passing the exam */
+		isPassingGrade: function(gradeStr) {
+			if (npu.getLanguage() == "hu") {
+				return gradeStr != "Elégtelen" && gradeStr != "Nem felelt meg" && gradeStr != "Nem jelent meg";
+			} else if (npu.getLanguage() == "en") {
+				return gradeStr != "Fail" && gradeStr != "Did not attend";
+			} else if (npu.getLanguage() == "de") {
+				// ???

```

Comment:
Apparently, the German version uses Hungarian grade names, so there is no need for the `de` branch.